controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -9921,8 +9921,14 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           name: provider.name,
           endpoint: provider.endpoint,
           loginURL: `${opts.webBaseUrl}/login?sso=${provider.alias}`,
-          signInRedirectURL: `${opts.keycloakApiUrl}/realms/${opts.keycloakRealm}/broker/${provider.alias}/endpoint`,
-          signOutRedirectURL: `${opts.keycloakApiUrl}/realms/${opts.keycloakRealm}/broker/${provider.alias}/endpoint/logout_response`,
+          signInRedirectURL: new URL(
```

Comment:
The proto defines a string. Tsc correctly infers that it is stringified?

Reply:
You mean does tsc understand that its a string?

Reply:
Yep, i do toString()